{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "execution": {
     "iopub.execute_input": "2022-01-25T22:48:39.837947Z",
     "iopub.status.busy": "2022-01-25T22:48:39.836977Z",
     "iopub.status.idle": "2022-01-25T22:48:42.160772Z",
     "shell.execute_reply": "2022-01-25T22:48:42.160215Z",
     "shell.execute_reply.started": "2022-01-25T22:48:39.837439Z"
    },
    "tags": []
   },
   "outputs": [],
   "source": [
    "import os, gc, time\n",
    "import numpy as np\n",
    "import pandas as pd\n",
    "import xarray as xr\n",
    "import matplotlib.pyplot as plt\n",
    "import matplotlib as mpl\n",
    "username = os.getenv('USER')\n",
    "import dask.array as da\n",
    "from dask_jobqueue import SLURMCluster\n",
    "from dask.distributed import Client, progress\n",
    "import dask\n",
    "from collections import OrderedDict\n",
    "dask.config.set(**{'array.slicing.split_large_chunks': False})\n",
    "# dask.config.set({\"distributed.workers.memory.target\": 4})\n",
    "\n",
    "from dscim import ProWaiter\n",
    "from dscim.menu.simple_storage import StackedDamages, EconVars, Climate\n",
    "from dscim.menu.risk_aversion import RiskAversionRecipe\n",
    "from dscim.menu.baseline import Baseline\n",
    "from dscim.menu.equity import EquityRecipe\n",
    "from itertools import product, chain\n",
    "import itertools\n",
    "# import seaborn as sns\n",
    "from dscim.fair.utils import modeler\n",
    "from dscim.fair.estimations import c_equivalence\n",
    "from functools import reduce\n",
    "from pathlib import Path\n",
    "\n",
    "import seaborn as sns\n",
    "from dscim.fair.estimations import compute_damages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "execution": {
     "iopub.execute_input": "2022-01-25T22:48:42.161748Z",
     "iopub.status.busy": "2022-01-25T22:48:42.161586Z",
     "iopub.status.idle": "2022-01-25T22:48:42.166424Z",
     "shell.execute_reply": "2022-01-25T22:48:42.165919Z",
     "shell.execute_reply.started": "2022-01-25T22:48:42.161726Z"
    },
    "tags": []
   },
   "outputs": [],
   "source": [
    "discount_types = [\"euler_gwr\", \"constant\", \"euler_ramsey\"] # , \"naive_gwr\", \"naive_ramsey\"]\n",
    "menu_options = [\"risk_aversion\", \"adding_up\", \"equity\"]\n",
    "pulse_years = [2020]\n",
    "# pulse_years = list(range(2025, 2051, 5))\n",
    "weitzman_values = [x / 10.0 for x in range(1, 11, 1)] + [0.25, 0.01, 0.001, 0.0001]\n",
    "global_cons = False\n",
    "masks = [\"unmasked\",\n",
    "         \"truncate_at_ecs995symmetric_passing_mask\",\n",
    "        \"truncate_at_ecs990symmetric_passing_mask\",\n",
    "        \"truncate_at_ecs950symmetric_passing_mask\",\n",
    "        \"truncate_at_ecs830symmetric_passing_mask\",\n",
    "        \"truncate_at_ecs750symmetric_passing_mask\",\n",
    "        ]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# run all"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "execution": {
     "iopub.execute_input": "2022-01-25T22:48:42.167707Z",
     "iopub.status.busy": "2022-01-25T22:48:42.167460Z",
     "iopub.status.idle": "2022-01-25T22:48:42.171315Z",
     "shell.execute_reply": "2022-01-25T22:48:42.170836Z",
     "shell.execute_reply.started": "2022-01-25T22:48:42.167678Z"
    },
    "tags": []
   },
   "outputs": [],
   "source": [
    "sectors = [\"integration_CAMEL_clipped\"] # \"integration_AMEL_clipped\", \"coastal\", \n",
    "AR_list = [5,6]\n",
    "combos = [(s, p, d, m, ar, ma) \n",
    "          for p in pulse_years\n",
    "          for s in sectors \n",
    "          for d in discount_types \n",
    "          for m in menu_options\n",
    "          for ar in AR_list\n",
    "          for ma in masks\n",
    "         ]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "execution": {
     "iopub.execute_input": "2022-01-25T22:48:42.172976Z",
     "iopub.status.busy": "2022-01-25T22:48:42.172720Z"
    },
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "=========================================================\n",
      "('integration_CAMEL_clipped', 2020, 'euler_gwr', 'risk_aversion', 5, 'unmasked')\n",
      "\n",
      " Executing Running risk_aversion class with the following parameters: \n",
      "Sector: integration_CAMEL_clipped \n",
      "Path to damages: None \n",
      "Menu recipe: risk_aversion \n",
      "Damages delta: summed_delta \n",
      "Discount level: global \n",
      "Discount type: euler_gwr \n",
      "Damage function path: /mnt/CIL_integration/damage_function_library/damage_function_library_integration/integration_CAMEL_clipped \n",
      "\n",
      "\n",
      "Processing damage functions ...\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Processing SCC calculation ...\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Extrapolating global consumption.\n",
      "End-of-century growth rates are not capped.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/xarray/core/nanops.py:148: RuntimeWarning: Mean of empty slice\n",
      "  return np.nanmean(a, axis=axis, dtype=dtype)\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/xarray/core/nanops.py:148: RuntimeWarning: Mean of empty slice\n",
      "  return np.nanmean(a, axis=axis, dtype=dtype)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "continuous discounting\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_discount_factors.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Skipped saving.\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_marginal_damages.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_uncollapsed_sccs.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_scc.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Results available: /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/unmasked/\n",
      "remaining combos: 107\n",
      "=========================================================\n",
      "('integration_CAMEL_clipped', 2020, 'euler_gwr', 'risk_aversion', 5, 'truncate_at_ecs995symmetric_passing_mask')\n",
      "\n",
      " Executing Running risk_aversion class with the following parameters: \n",
      "Sector: integration_CAMEL_clipped \n",
      "Path to damages: None \n",
      "Menu recipe: risk_aversion \n",
      "Damages delta: summed_delta \n",
      "Discount level: global \n",
      "Discount type: euler_gwr \n",
      "Damage function path: /mnt/CIL_integration/damage_function_library/damage_function_library_integration/integration_CAMEL_clipped \n",
      "\n",
      "\n",
      "Processing damage functions ...\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs995symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Processing SCC calculation ...\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs995symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Extrapolating global consumption.\n",
      "End-of-century growth rates are not capped.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "Skipped saving.\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs995symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "continuous discounting\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs995symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_discount_factors.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs995symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Skipped saving.\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs995symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs995symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs995symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_marginal_damages.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs995symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs995symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_uncollapsed_sccs.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs995symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_scc.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Results available: /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs995symmetric_passing_mask/\n",
      "remaining combos: 106\n",
      "=========================================================\n",
      "('integration_CAMEL_clipped', 2020, 'euler_gwr', 'risk_aversion', 5, 'truncate_at_ecs990symmetric_passing_mask')\n",
      "\n",
      " Executing Running risk_aversion class with the following parameters: \n",
      "Sector: integration_CAMEL_clipped \n",
      "Path to damages: None \n",
      "Menu recipe: risk_aversion \n",
      "Damages delta: summed_delta \n",
      "Discount level: global \n",
      "Discount type: euler_gwr \n",
      "Damage function path: /mnt/CIL_integration/damage_function_library/damage_function_library_integration/integration_CAMEL_clipped \n",
      "\n",
      "\n",
      "Processing damage functions ...\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs990symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Processing SCC calculation ...\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs990symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Extrapolating global consumption.\n",
      "End-of-century growth rates are not capped.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "Skipped saving.\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs990symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "continuous discounting\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs990symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_discount_factors.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs990symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Skipped saving.\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs990symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs990symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs990symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_marginal_damages.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs990symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs990symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_uncollapsed_sccs.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs990symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_scc.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Results available: /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs990symmetric_passing_mask/\n",
      "remaining combos: 105\n",
      "=========================================================\n",
      "('integration_CAMEL_clipped', 2020, 'euler_gwr', 'risk_aversion', 5, 'truncate_at_ecs950symmetric_passing_mask')\n",
      "\n",
      " Executing Running risk_aversion class with the following parameters: \n",
      "Sector: integration_CAMEL_clipped \n",
      "Path to damages: None \n",
      "Menu recipe: risk_aversion \n",
      "Damages delta: summed_delta \n",
      "Discount level: global \n",
      "Discount type: euler_gwr \n",
      "Damage function path: /mnt/CIL_integration/damage_function_library/damage_function_library_integration/integration_CAMEL_clipped \n",
      "\n",
      "\n",
      "Processing damage functions ...\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs950symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Processing SCC calculation ...\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs950symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Extrapolating global consumption.\n",
      "End-of-century growth rates are not capped.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "Skipped saving.\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs950symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "continuous discounting\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs950symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_discount_factors.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs950symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Skipped saving.\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs950symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs950symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs950symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_marginal_damages.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs950symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs950symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_uncollapsed_sccs.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs950symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_scc.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Results available: /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs950symmetric_passing_mask/\n",
      "remaining combos: 104\n",
      "=========================================================\n",
      "('integration_CAMEL_clipped', 2020, 'euler_gwr', 'risk_aversion', 5, 'truncate_at_ecs830symmetric_passing_mask')\n",
      "\n",
      " Executing Running risk_aversion class with the following parameters: \n",
      "Sector: integration_CAMEL_clipped \n",
      "Path to damages: None \n",
      "Menu recipe: risk_aversion \n",
      "Damages delta: summed_delta \n",
      "Discount level: global \n",
      "Discount type: euler_gwr \n",
      "Damage function path: /mnt/CIL_integration/damage_function_library/damage_function_library_integration/integration_CAMEL_clipped \n",
      "\n",
      "\n",
      "Processing damage functions ...\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs830symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Processing SCC calculation ...\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs830symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Extrapolating global consumption.\n",
      "End-of-century growth rates are not capped.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "Skipped saving.\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs830symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "continuous discounting\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs830symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_discount_factors.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs830symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Skipped saving.\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs830symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs830symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs830symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_marginal_damages.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs830symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs830symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_uncollapsed_sccs.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs830symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_scc.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Results available: /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs830symmetric_passing_mask/\n",
      "remaining combos: 103\n",
      "=========================================================\n",
      "('integration_CAMEL_clipped', 2020, 'euler_gwr', 'risk_aversion', 5, 'truncate_at_ecs750symmetric_passing_mask')\n",
      "\n",
      " Executing Running risk_aversion class with the following parameters: \n",
      "Sector: integration_CAMEL_clipped \n",
      "Path to damages: None \n",
      "Menu recipe: risk_aversion \n",
      "Damages delta: summed_delta \n",
      "Discount level: global \n",
      "Discount type: euler_gwr \n",
      "Damage function path: /mnt/CIL_integration/damage_function_library/damage_function_library_integration/integration_CAMEL_clipped \n",
      "\n",
      "\n",
      "Processing damage functions ...\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs750symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Processing SCC calculation ...\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs750symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Extrapolating global consumption.\n",
      "End-of-century growth rates are not capped.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "Skipped saving.\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs750symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "continuous discounting\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs750symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_discount_factors.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs750symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Skipped saving.\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs750symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs750symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs750symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_marginal_damages.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs750symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs750symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_uncollapsed_sccs.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saving /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs750symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_scc.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Results available: /mnt/CIL_integration/menu_results_AR5_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs750symmetric_passing_mask/\n",
      "remaining combos: 102\n",
      "=========================================================\n",
      "('integration_CAMEL_clipped', 2020, 'euler_gwr', 'risk_aversion', 6, 'unmasked')\n",
      "\n",
      " Executing Running risk_aversion class with the following parameters: \n",
      "Sector: integration_CAMEL_clipped \n",
      "Path to damages: None \n",
      "Menu recipe: risk_aversion \n",
      "Damages delta: summed_delta \n",
      "Discount level: global \n",
      "Discount type: euler_gwr \n",
      "Damage function path: /mnt/CIL_integration/damage_function_library/damage_function_library_integration/integration_CAMEL_clipped \n",
      "\n",
      "\n",
      "Processing damage functions ...\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR6_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Processing SCC calculation ...\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR6_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Extrapolating global consumption.\n",
      "End-of-century growth rates are not capped.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "Skipped saving.\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR6_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "continuous discounting\n",
      "Saving /mnt/CIL_integration/menu_results_AR6_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_discount_factors.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR6_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Skipped saving.\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR6_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR6_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Saving /mnt/CIL_integration/menu_results_AR6_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_marginal_damages.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR6_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "Saving /mnt/CIL_integration/menu_results_AR6_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_uncollapsed_sccs.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saving /mnt/CIL_integration/menu_results_AR6_bc39_integration/integration_CAMEL_clipped/2020/unmasked/risk_aversion_euler_gwr_eta2.0_rho0.0_scc.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Results available: /mnt/CIL_integration/menu_results_AR6_bc39_integration/integration_CAMEL_clipped/2020/unmasked/\n",
      "remaining combos: 101\n",
      "=========================================================\n",
      "('integration_CAMEL_clipped', 2020, 'euler_gwr', 'risk_aversion', 6, 'truncate_at_ecs995symmetric_passing_mask')\n",
      "\n",
      " Executing Running risk_aversion class with the following parameters: \n",
      "Sector: integration_CAMEL_clipped \n",
      "Path to damages: None \n",
      "Menu recipe: risk_aversion \n",
      "Damages delta: summed_delta \n",
      "Discount level: global \n",
      "Discount type: euler_gwr \n",
      "Damage function path: /mnt/CIL_integration/damage_function_library/damage_function_library_integration/integration_CAMEL_clipped \n",
      "\n",
      "\n",
      "Processing damage functions ...\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR6_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs995symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Processing SCC calculation ...\n",
      "Masking anomalies\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR6_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs995symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Extrapolating global consumption.\n",
      "End-of-century growth rates are not capped.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "Skipped saving.\n",
      "continuous discounting\n",
      "continuous discounting\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR6_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs995symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "continuous discounting\n",
      "Saving /mnt/CIL_integration/menu_results_AR6_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs995symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_discount_factors.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipped saving.\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR6_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs995symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Skipped saving.\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR6_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs995symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "You have passed a path to damage functions. Loading damage function coefficients...\n",
      "Saving /mnt/CIL_integration/menu_results_AR6_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs995symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_damage_function_coefficients.nc4\n",
      "Saving /mnt/CIL_integration/menu_results_AR6_bc39_integration/integration_CAMEL_clipped/2020/truncate_at_ecs995symmetric_passing_mask/risk_aversion_euler_gwr_eta2.0_rho0.0_marginal_damages.nc4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n",
      "/home/kschwarz/miniconda3/envs/dscim/lib/python3.8/site-packages/dask/core.py:121: RuntimeWarning: invalid value encountered in true_divide\n",
      "  return func(*(_execute_task(a, cache) for a in args))\n"
     ]
    }
   ],
   "source": [
    "n_combos = len(combos)\n",
    "\n",
    "while n_combos > 0: \n",
    "    combo = combos[0]\n",
    "    print(\"=========================================================\")\n",
    "    print(combo)\n",
    "    sector = combo[0]\n",
    "    pulse_year = combo[1]\n",
    "    discount_type = combo[2]\n",
    "    menu_option = combo[3]\n",
    "    AR = combo[4]\n",
    "    mask = combo[5]\n",
    "\n",
    "    w = ProWaiter(path_to_config=f'/home/{username}/repos/integration/configs/epa_tool_config-histclim_AR{AR}.yaml')\n",
    "    save_path=f'/mnt/CIL_integration/menu_results_AR{AR}_bc39_integration/{sector}/{pulse_year}/{mask}/'\n",
    "\n",
    "    if mask=='unmasked':\n",
    "        mask = None\n",
    "\n",
    "    kwargs = {'discounting_type' : discount_type,\n",
    "              'sector': sector,\n",
    "              'ce_path' : f'/shares/gcp/integration/CE_library_epa_vsl_bc39/{sector}/',\n",
    "              'save_path' : save_path,\n",
    "              'weitzman_parameter' : weitzman_values,\n",
    "              'pulse_year' : pulse_year,\n",
    "              'ecs_mask_name' : mask\n",
    "             }\n",
    "\n",
    "    if \"CAMEL\" in sector:\n",
    "        kwargs.update({\n",
    "            'damage_function_path' : f'/mnt/CIL_integration/damage_function_library/damage_function_library_integration/{sector}'})\n",
    "\n",
    "    menu_item = w.menu_factory(menu_key=menu_option,\n",
    "                                           sector=sector,\n",
    "                                           kwargs=kwargs\n",
    "                                          )\n",
    "    menu_item.order_plate('scc')\n",
    "    if \"CAMEL\" not in sector:\n",
    "        menu_item.damage_function_fit\n",
    "        \n",
    "    if global_cons == True:\n",
    "        menu_item.global_consumption_no_pulse.to_netcdf(\n",
    "            f\"{save_path}/{menu_option}_{discount_type}_eta{menu_item.eta}_rho{menu_item.rho}_global_consumption_no_pulse.nc4\")\n",
    "\n",
    "    combos.remove(combo)\n",
    "    n_combos = len(combos)\n",
    "    print(f\"remaining combos: {n_combos}\")\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
